src/api/APIFactory.js

```diff
@@ -510,6 +516,21 @@ class APIFactory {
         this.appIntegrationsAPI = new AppIntegrationsAPI(this.options);
         return this.appIntegrationsAPI;
     }
+
+    /**
+     * API for Box Edit
+     *
+     * @param {boolean} shouldDestroy - true if the factory should destroy before returning the call
+     * @return {BoxEditAPI} BoxEditAPI instance
+     */
+    getBoxEditAPI(shouldDestroy: boolean): BoxEditAPI {
+        if (shouldDestroy) {
```

Comment:
Seems a little odd to pass this in but not destroy the BoxEditAPI itself. Maybe just omit the param entirely?